components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIProvider.java

```diff
@@ -1761,6 +1761,7 @@ Mediation updateApiSpecificMediationPolicyContent(String apiId, Mediation mediat
      * @throws APIManagementException
      */
     void saveAsyncApiDefinition(API api, String jsonText) throws APIManagementException;
+
```

Comment:
Remove this empty line

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/10626